File: java/src/n01_oop_vererbung/Mitarbeiter.java
package n01_oop_vererbung;

public class Mitarbeiter extends Person {
    public int personalNummer;
    public boolean fuehrungskraft;
    public double gehalt;

    public Mitarbeiter(String name, String gebOrt, String gebDat, int personalNummer, boolean fuehrungskraft, double gehalt) {
        super(name, gebOrt, gebDat);
        this.personalNummer = personalNummer;
        this.fuehrungskraft = fuehrungskraft;
        this.gehalt = gehalt;
    }
}

File: dart/src/n01_oop_vererbung/mitarbeiter.dart
import 'person.dart';

class Mitarbeiter extends Person {
  Mitarbeiter(super.name, super.gebOrt, super.gebDat, this.personalNummer, this.fuehrungskraft, this.gehalt);

  final int personalNummer;
  final bool fuehrungskraft;
  final double gehalt;
}
